<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>553</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hantek 6022BL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>-10</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnGet">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>ARM</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_rise">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RISE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FALL</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AUTO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NORMAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SINGLE</string>
      </property>
     </item>
    </widget>
    <widget class="QDial" name="dialTrigger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCHSel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CH1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Level</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelTrigger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00V</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelCurX12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QDial" name="dialHoldoff">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDial" name="dialDelay">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>500</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Holdoff</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblholdoff">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>40.00ms</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblfreq">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00us</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>270</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
    <widget class="QComboBox" name="comboBoxV1div">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>2 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200mV/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100mV/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50mV/div</string>
      </property>
     </item>
    </widget>
    <widget class="QDial" name="dialCursorV1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH1ON">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CH1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH1Glitch">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Glitch</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxMode1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Position</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gain</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH1Add">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH1Inv">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>INV</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCH1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.000V</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QComboBox" name="comboBoxV2div">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>2 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5 V/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200mV/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100mV/div</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50mV/div</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH2ON">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CH2</string>
     </property>
    </widget>
    <widget class="QDial" name="dialCursorV2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH2Glitch">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Glitch</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCH2Inv">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>INV</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxMode2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Position</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gain</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblCH2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.000V</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboSampling">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentText">
     <string>20ns</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxVisibleItems">
     <number>12</number>
    </property>
    <item>
     <property name="text">
      <string>20ns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50ns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100ns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200ns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>400ns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100ms</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_to_file"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOffset_Null"/>
    <addaction name="actionSetScaleFactor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_to_file">
   <property name="text">
    <string>Save to file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSetScaleFactor">
   <property name="text">
    <string>Set Scale</string>
   </property>
  </action>
  <action name="actionOffset_Null">
   <property name="text">
    <string>Offset Null</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
